api/envoy/config/grpc_credential/v2alpha/aws_iam.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+// [#protodoc-title: Grpc Credentials AWS IAM]
+// Configuration for AWS IAM Grpc Credentials Plugin
+
+package envoy.config.grpc_credential.v2alpha;
+
+option java_outer_classname = "AwsIamProto";
+option java_package = "io.envoyproxy.envoy.config.grpc_credential.v2alpha";
+option java_multiple_files = true;
+option go_package = "v2alpha";
+
+import "validate/validate.proto";
+
+message AwsIamConfig {
+  // The `service namespace
+  // <https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-aws-service-namespaces>`_
+  // of the Grpc endpoint.
+  //
+  // Example: appmesh
+  string service_name = 1 [(validate.rules).string.min_bytes = 1];
+
+  // The `region <https://docs.aws.amazon.com/general/latest/gr/rande.html>`_ hosting the Grpc
+  // endpoint.
+  //
+  // Example: us-west-2
+  string region = 2;
```

Comment:
Can you document what is done when this is empty? From looking at the the code it looks like it tries to pull it from the env?

Reply:
correct. documented.